### PR TITLE
UPDATE on the product descriptions

### PR DESCRIPTION
**Pull request by EjazaliRezayi**
I corrected the display and positions of text/image on almost all (except cheesedip/chips).
This time it should be all good.
I'll fix chips and cheesedips later.
Also, we need correct images in ice cream and popsicle.

### CODE REVIEW

**Review by georgesgrondin — CHANGES_REQUESTED** [review]
Need to fix some of the 'back to aisle' links - a few of them point to bread

**Comment by sarzhou** [comment]
> ### ###

File: product-descriptions/icecream.html [hunk 1]
```diff
@@ -13,31 +13,30 @@
...
     <header>
         <div class="product-name-header">
             Product Description - Ice Cream, 473 mL
         </div>
-
+    </header>
+    <nav>
+        <ul>
+            <li><a href="../index.html">Home Page</a></li>
+            <li><a href="../aisles/bread.html">Return to Aisle</a></li>
```
Comment (georgesgrondin) [inline product-descriptions/icecream.html:25]:
```suggestion
            <li><a href="../aisles/bread.html">Return to Aisle</a></li>

```

Comment (georgesgrondin) [inline product-descriptions/icecream.html:25]:
```suggestion
            <li><a href="../aisles/bread.html">Return to Aisle</a></li>

```

**Review by sarzhou — APPROVED** [review]

**Pull request merged**